deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -813,6 +814,22 @@ def modify_metadata_vuldet_feed(feed, timestamp):
     control_service('start', daemon='wazuh-db')
 
 
+def modify_nvd_metadata_vuldet(timestamp):
+    """Function to modify the timestamp value of the nvd_metadata table.
+
+    Args:
+        timestamp (int): The new timestamp value to set.
+    """
+    query_string = f"UPDATE NVD_METADATA SET LAST_UPDATE={timestamp};"
+    for _ in range(VULN_DETECTOR_GLOBAL_TIMEOUT):
+        try:
+            make_query(CVE_DB_PATH, [query_string])
+            return
+        except sqlite3.OperationalError:
+            sleep(1)
+    raise sqlite3.OperationalError
```

Comment:
```suggestion
    for _ in range(VULN_DETECTOR_GLOBAL_TIMEOUT):
        try:
            make_query(CVE_DB_PATH, [query_string])
            break
        except sqlite3.OperationalError:
            sleep(1)
    else:
        raise sqlite3.OperationalError
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1428/commits/1d7710d1cf9ff09af486d8ffdeb43e0b4dce8a9b